.github/workflows/release.yml

```diff
@@ -63,3 +63,65 @@ jobs:
           gh release upload --clobber "${{ github.ref_name }}" \
             Docs.zip \
             Docs.zip.sha{1,256}sum
+      - name: Upload Precompiled to S3
```

Comment:
I am thinking we should keep those separate:

1. We can move this to the repo.hex.pm task
2. We can change repo.hex.pm to run on tags too

WDYT?

The idea is to keep this only specific to GitHub and the other one to Hex.

Reply:
https://github.com/elixir-lang/elixir/pull/12242/commits/3a18d8270e7866d2ed1795ad3715f4bb797a545b